Common/Data/SubscriptionManager.cs

```diff
@@ -180,7 +183,27 @@ public IReadOnlyList<TickType> GetDataTypesForSecurity(SecurityType securityType
             return AvailableDataTypes[securityType];
         }
 
+        /// <summary>
+        /// Get the data feed types for a given <see cref="SecurityType"/> <see cref="Resolution"/> 
+        /// </summary>
+        /// <param name="symbolSecurityType">The <see cref="SecurityType"/> used to determine the types</param>
+        /// <param name="resolution">The resolution of the data requested</param>
+        /// <param name="isCanonical">Indicates whether the security is Canonical (future and options)</param>
+        /// <returns>Types that should be added to the <see cref="SubscriptionDataConfig"/></returns>
+        public IEnumerable<Type> LookupSubscriptionConfigDataTypes(SecurityType symbolSecurityType, Resolution resolution, bool isCanonical)
+        {
+            if (isCanonical)
+            {
+                return new List<Type>() {typeof(ZipEntryName)};
+            }
 
+            if (resolution == Resolution.Tick)
+            {
+                return new List<Type>() { typeof(Tick) };
```

Comment:
Is this valid for `SecurityType.Equity.Base`? 
If this method is not intended to be called with that argument then maybe a guard clause to check and throw would be helpful.